tests/functional/spec/features/unit_label_hover/unit_label_hover.spec.js

```diff
@@ -0,0 +1,11 @@
+describe("Unit Label Hover", () => {
```

Comment:
Add it to existing unit spec.

Reply:
Added now.

Reply:
As I mentioned, length units were all American en.